engine/crates/engine-v2/src/sources/graphql/mod.rs

```diff
@@ -94,12 +94,15 @@ pub(crate) struct GraphqlExecutor<'ctx, R: Runtime> {
 impl<'ctx, R: Runtime> GraphqlExecutor<'ctx, R> {
     #[tracing::instrument(skip_all)]
     pub async fn execute(self, mut response_part: ResponsePart) -> ExecutionResult<ResponsePart> {
-        let subgraph_request_span = SubgraphRequestSpan::new(self.subgraph.name())
-            .with_url(self.subgraph.url())
-            .with_operation_type(self.operation.ty.as_ref())
-            // The query string contains no input values, only variables. So it's safe to log.
-            .with_document(&self.operation.query)
-            .into_span();
+        let span = SubgraphRequestSpan {
+            name: self.subgraph.name(),
+            operation_type: self.operation.ty.as_str(),
+            // The generated query does not contain any data, everything are in the variables, so
+            // it's safe to use.
+            sanitized_query: &self.operation.query,
```

Comment:
Sam here, should we maybe consider a different name here because the query is not really sanitized?

Reply:
same thing